_sass/blocks/_communities_content.scss

```diff
@@ -7,15 +7,14 @@
 .communities_content {
   position: relative;
   ul {
+    @include heading('para-sm');
 
-    @include heading("para-sm");
-    
-  	list-style-type: square;
     font-weight: $weight-light;
-	  padding-left: $base-padding;
-	  padding-bottom: $base-padding;
+    list-style-type: square;
+    padding-left: $base-padding;
```

Comment:
Properties should be ordered font-weight, list-style-type, padding-bottom, padding-left